src/oas3/__tests__/operation.test.ts

```diff
@@ -455,7 +455,7 @@ describe('transformOas3Operation', () => {
       {
         description: void 0,
         url: 'spec/server',
-        variables: {},
+        variables: void 0,
```

Comment:
You can remove this property check entirely
```suggestion
```

and the same goes for all the properties above.